# get Force      : = 0
# get Position 1 : = 1
# get Position 2 : = 2

int32 get_target 
---
int32 return_data